lib/ansible/modules/cloud/digital_ocean/digital_ocean_certificate_facts.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2018, Ansible Project
+# Copyright: (c) 2018, Abhijeet Kasurde <akasurde@redhat.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+
+DOCUMENTATION = '''
+---
+module: digital_ocean_certificate_facts
+short_description: Gather facts about DigitalOcean certificates
+description:
+    - This module can be used to gather facts about DigitalOcean provided certificates.
+author: "Abhijeet Kasurde (@akasurde)"
+version_added: "2.6"
+options:
+  certificate_id:
+    description:
+     - Certificate ID that can be used to identify and reference a certificate.
+    required: false
+requirements:
+  - "python >= 2.6"
+extends_documentation_fragment: digital_ocean.documentation
+'''
+
+
+EXAMPLES = '''
+- name: Gather facts about all certificates
+  digital_ocean_certificate_facts:
+    oauth_token: "{{ oauth_token }}"
+
+- name: Gather facts about certificate with given id
+  digital_ocean_certificate_facts:
+    oauth_token: "{{ oauth_token }}"
+    certificate_id: "892071a0-bb95-49bc-8021-3afd67a210bf"
+
+- name: Get not after facts about certificate
```

Comment:
Would a better example be the use of the `sha1_fingerprint` or `id`?

Reply:
OK.

Reply:
I'm just trying to think of a simple use case for these facts.

Reply:
Use case can be 
- identify existing certificates in Digital Ocean
- check expiration of certificates and automation related to that
- check validity of certificates